icubam/backoffice/handlers/operational_dashboard.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import tornado.web
+import math
+from collections import defaultdict
+
+from bokeh.embed import components
+from bokeh.plotting import figure
+from bokeh.transform import dodge
+from bokeh.models import ColumnDataSource
+
+from icubam.db.store import to_pandas
+from icubam.backoffice.handlers import base
+
+
+class OperationalDashHandler(base.BaseHandler):
+  ROUTE = '/operational-dashboard'
+
+  @tornado.web.authenticated
+  async def get(self):
+    """Serves a page with a table gathering current bedcount data with some extra information."""
+    arg_region = self.get_query_argument('region', default=None)
+    current_user = self.get_current_user()
+
+    current_region = None
+    current_region_name = "Toutes les régions"
+    if arg_region is not None and arg_region.isdigit():
+      try:
+        current_region = self.db.get_region(int(arg_region))
+        current_region_name = current_region.name
+      except Exception:
+        logging.warning(
+          f"Invalid query argument: region={arg_region} "
+          f"Falling back to all regions."
+        )
+
+    figures = []
+
+    bed_counts = self.db.get_visible_bed_counts_for_user(current_user.user_id)
+
+    bed_counts = to_pandas(bed_counts)
+
+    if current_region is not None:
+      mask = bed_counts['icu_region_id'] == current_region.region_id
```

Comment:
i would check that,  `if not self.user.is_admin`  then we should make sure that the requested region_id is part of the region_ids of the managed_icus.

Reply:
we do not want the a user to have access to regions he does not belong